source/how-tos/app-development/interactive/form.rst

```diff
@@ -505,6 +505,20 @@ are:
 
           cacheable: false
 
+.. describe:: display (Boolean, false)
+
+     whether the form item should be displayed in the session card after it was created.
+
+     Default
+       False. The form item will not be displayed. 
```

Comment:
Can you add the `yaml` block here too? IDK why I think it just sort of adds to uniformity to have yaml blocks for both default and example.

Reply:
Fixed.